### ISSUE TITLE
Missing code that will install glusterfs daemon.

### ISSUE THREAD

**Issue by hetii** [gluster/gluster-ansible-cluster#20]
Hi, I try to deploy glusterfs in raw machines but I got: 

    TASK [gluster.cluster/roles/gluster_volume : Start glusterd on the nodes if not already started] *************************************************************************************************************************************************************
    failed: [devops-env1-glusterfs-zone1-01 -> 10.11.1.150] (item=devops-env1-glusterfs-zone1-01) => {"changed": false, "item": "devops-env1-glusterfs-zone1-01", "msg": "Could not find the requested service glusterd: host"}

Do I need to create manually ansible task to install it?

**Comment by sac** [comment]
> Hi, I try to deploy glusterfs in raw machines but I got:
> 
> ```
> TASK [gluster.cluster/roles/gluster_volume : Start glusterd on the nodes if not already started] *************************************************************************************************************************************************************
> failed: [devops-env1-glusterfs-zone1-01 -> 10.11.1.150] (item=devops-env1-glusterfs-zone1-01) => {"changed": false, "item": "devops-env1-glusterfs-zone1-01", "msg": "Could not find the requested service glusterd: host"}
> ```
> 
> Do I need to create manually ansible task to install it?

No, you have to call the role gluster.repolsitories. That should take care of installing
the packages. Let me know if you face any issues there. I can help with that.

https://github.com/gluster/gluster-ansible-repositories/
